generators/lib/prompts.js

```diff
@@ -66,6 +65,27 @@ var prompts = [
     choices: function(answers) {
       return answers['drupalDistro'].versions;
     }
+  },
+  {
+    type: 'confirm',
+    name: 'doGenerateProfile',
+    message: 'Do you want to generate a custom Drupal profile?',
+    default: false
```

Comment:
Since this is for Phase2 best practices, default should be "true" otherwise the config workflow doesn't work.

Reply:
Fair enough. I was pretty sure that wouldn't fly. I did it as a cheat to keep the test changes here and in outrigger-drupal simpler, but it's not that much more work.